src/PKSim.Core/Model/IndividualProteinTask.cs

```diff
@@ -0,0 +1,273 @@
+      {
+         foreach (var container in simulationSubject.Organism.NonGITissueContainers)
+         {
+            addTissueParameters(container, moleculeName, CoreConstants.Groups.ORGANS_AND_TISSUES);
+         }
+
+         foreach (var organ in simulationSubject.Organism.GITissueContainers)
+         {
+            addTissueParameters(organ, moleculeName, CoreConstants.Groups.GI_NON_MUCOSA_TISSUE);
+         }
+      }
+
+      private void addTissueParameters(IContainer organ, string moleculeName, string groupName)
+      {
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.BloodCells), organ.Name, moleculeName, groupName,
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_BLOOD_CELLS)
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Plasma), organ.Name, moleculeName, groupName,
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_PLASMA)
+         );
+
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Endosome), organ.Name, moleculeName, groupName,
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_ENDOSOME)
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Intracellular), organ.Name, moleculeName, groupName,
+            relExpParam(REL_EXP),
+            fractionParam(FRACTION_EXPRESSED_INTRACELLULAR, CoreConstants.Rate.PARAM_F_EXP_INTRACELLULAR, editable: false),
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTRACELLULAR)
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Interstitial), organ.Name, moleculeName, groupName,
+            fractionParam(FRACTION_EXPRESSED_INTERSTITIAL, CoreConstants.Rate.ZERO_RATE),
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTERSTITIAL)
+         );
+      }
+
+      private ParameterValueMetaData relExpParam(string paramName) => new ParameterValueMetaData
+      {
+         ParameterName = paramName,
+         Dimension = DIMENSIONLESS,
+         DefaultValue = 0,
+         GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         BuildingBlockType = PKSimBuildingBlockType.Individual,
+         IsInput =  true,
+      };
+
+      private ParameterRateMetaData fractionParam(string paramName, string rate, string groupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         bool editable = true) =>
+         new ParameterRateMetaData
+         {
+            ParameterName = paramName,
+            Rate = rate,
+            CalculationMethod = CoreConstants.CalculationMethod.EXPRESSION_PARAMETERS,
+            BuildingBlockType = PKSimBuildingBlockType.Individual,
+            CanBeVaried = true,
+            CanBeVariedInPopulation = true,
+            ReadOnly = !editable,
+            Dimension = CoreConstants.Dimension.Fraction,
+            GroupName = groupName,
+            IsInput = true
```

Comment:
@msevestre 
I think `IsInput `must be **false** for the non-constant formula parameters (like f_expression_cell etc.).
THus if  `Rate `is not _Zero_Rate_ or _One_Rate_.

Reply:
I don't even know. I think for zero parameter it should also be false. I think we have to think about it